mks/admin.py

```diff
@@ -93,6 +95,16 @@ def change_view(self, request, object_id, extra_context=None):
     def queryset(self, request):
         return super(MemberAdmin, self).queryset(
             request).select_related('current_party')
+
+    def save_model(self, request, obj, form, change):
+        # Delete the cache of the current member after updating:
+        cache.delete('mk_%d' % obj.id)
+        # Delete the template key
+        #key = make_template_fragment_key('mks_detail', [obj.id])
+        #cache.delete(key) # invalidates cached template fragment
+        super(MemberAdmin, self).save_model(request, obj, form, change)
```

Comment:
@wasymur - the following code should clear the most common cases (maybe even 100% of cases..):
* key = make_template_fragment_key('mks_detail', [obj.id, 1, obj.get_absolute_url()])